### PR TITLE
Show the status file of crowbar upgrade (if it exists)

### PR DESCRIPTION
**Pull request by jsuchome**

### CODE REVIEW

File: suse_openstack_cloud [hunk 1]
```diff
@@ -260,6 +260,12 @@ echo; echo
 find_and_pconf_files /srv/tftpboot/discovery/pxelinux.cfg -type f
 find_and_pconf_files /srv/tftpboot/nodes                  -type f
 
+echo; echo
+find                 /var/lib/crowbar/upgrade -ls
+if ls /va/lib/crowbar/upgrade/*progress.yml >/dev/null 2>&1 ; then
```
Comment (aspiers) [inline suse_openstack_cloud]: Typo - has this been tested yet?

Comment (jsuchome) [reply suse_openstack_cloud]: Oh. (I tested it manually, then obviously broken it)

**Comment by jsuchome** [comment]
wait a bit, it's somehow not working...

**Comment by jsuchome** [comment]
OK, I've updated it now, I did not realize `find_and_pconf_files` needs a directory....

**Pull request merged**